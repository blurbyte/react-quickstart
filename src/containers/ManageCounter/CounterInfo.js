import styled from 'styled-components';

const CounterInfo = styled.div`
  display: flex;
  justify-content: space-between;
`;

export default CounterInfo;
