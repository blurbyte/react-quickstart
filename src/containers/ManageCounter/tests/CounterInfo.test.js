import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import CounterInfo from '../CounterInfo';

describe('<CounterInfo />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <CounterInfo />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
