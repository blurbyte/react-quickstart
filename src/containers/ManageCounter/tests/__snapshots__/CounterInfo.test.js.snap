// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CounterInfo /> should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-between;
  -webkit-justify-content: space-between;
  -ms-flex-pack: space-between;
  justify-content: space-between;
}

<div
  className="c0"
/>
`;
